README.md

```diff
@@ -133,19 +133,40 @@ We have extended support for execution layer clients in simulated networks. Foll
 
 When it comes to consensus layer we currently support only `Prysm`.
 
+The easiest way to start a simulated network is to use a builder. It allows to configure the network in a fluent way and then start it. For example:
+
+```go
+builder := NewEthereumNetworkBuilder()
+cfg, err: = builder.
+    WithEthereumVersion(EthereumVersion_Eth2).
+    WithExecutionLayer(ExecutionLayer_Geth).
+    Build()
+```
+
+Since we support both `eth1` (aka pre-Merge) and `eth2` (aka post-Merge) client versions, you need to specify which one you want to use. You can do that by calling `WithEthereumVersion` method. There's no default provided. The only execption is when you use custom docker images (instead of default ones), because then we can determine which version it is based on the image version.
```

Comment:
Can we mention a scenario, or just generally why I would want to use one or the other?

Reply:
That is a very good question, why would you want to use this ethereum version or that one? I think the only reason is block speed.